lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -4,7 +4,7 @@
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.markup import HTML, Text
 from common.djangoapps.course_modes.models import CourseMode
-from common.djangoapps.util.course import should_display_grade
+from lms.djangoapps.certificates.models import CertificateStatuses
```

Comment:
Should we be importing this directly from the Certificates app `models.py`? Just thinking of this from the standpoint of [OEP-49](https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0049-django-app-patterns.rst).

Reply:
https://github.com/edx/edx-platform/pull/27842